readme.md

```diff
@@ -5,9 +5,11 @@ This repository is split into 3 components:
 - `client`: Contains the Axiom CLI interface and developer-facing AxiomCircuit exports
 - `circuit`: Handles lower-level interface with Wasm circuit libraries
 - `harness`: Test harness for `client`
+- `react`: Axiom React component for proving in React/Next.js dApps
```

Comment:
Should we call this "React component" or "React hook"?

Reply:
it technically exports a react component as well as a hook (for interacting w/ the component); will update the description accordingly